sdk/Lusid.Sdk/Utilities/ApiExceptionExtensions.cs

```diff
@@ -61,11 +64,26 @@ public static LusidProblemDetails ProblemDetails(this ApiException ex)
                 return null;
             }
 
-            //    ApiException.ErrorContent contains a JSON serialized ProblemDetails
-            return JsonConvert.DeserializeObject<LusidProblemDetails>(ex.ErrorContent.ToString(), new JsonConverter[]
+            try
             {
-                new PropertyBasedConverter(),
-            });
+                // ApiException.ErrorContent contains a JSON serialized ProblemDetails
+                return JsonConvert.DeserializeObject<LusidProblemDetails>(
+                    ex.ErrorContent.ToString(), new PropertyBasedConverter()
+                );
+            }
+            catch (JsonException)
+            {
+                return new LusidProblemDetails(
+                    name: "Invalid JSON error content response",
```

Comment:
I think a user might confuse this with the cause of the ApiException, I think this name should be the name of the `ApiException`. Perhaps the name of the HTTP Status code e.g. `InternalServerError`, `ServiceUnavailable` etc. would be a better fit.